qemu/tests/cfg/tcpreplay.cfg

```
@@ -1,8 +1,8 @@
 - tcpreplay:
     image_snapshot = yes
     type = tcpreplay
-    tcpreplay_file_name = "tcpreplay-4.3.1.tar.gz"
-    uncompress_dir = "tcpreplay-4.3.1"
+    tcpreplay_file_name = "tcpreplay-4.3.3.tar.gz"
+    uncompress_dir = "tcpreplay-4.3.3"
```

```suggestion
    uncompress_dir = "tcpreplay-4.3.3"
    tcpreplay_file_name = "${uncompress_dir}.tar.gz"
```
So that we can only modify one place in the future.